### ISSUE TITLE
Exemptions for containers

### ISSUE THREAD

**Issue by js-timbirkett** [FairwindsOps/polaris#408]
**Is your feature request related to a problem? Please describe.**
Some injected sidecar containers require privileges that violate some checks. For example, `istio-init` and `istio-proxy`. As we have very little control over these containers, it'd be great to be able to add exemptions to containers.

**Describe the solution you'd like**
Where a risk is known and accepted with a container within a pod, create an exception for it.

**Describe alternatives you've considered**
Adding an exception at the controller level but this would make it less obvious when a non-trusted container did something it shouldn't.

Please forgive me if I've missed something like the feature or capability already existing. It isn't clear from the documentation whether or not the functionality already exists.

**Comment by rbren** [comment]
Thanks for the request! This makes a lot of sense, and would be reasonably easy to implement.

**Comment by shaswa** [comment]
@js-timbirkett This is now addressed in #463 and should go out during next release!

**Comment by dbones** [comment]
Hello, has this been released?

I have the following:

```yaml
    - containerNames:
        - istio-init
        - istio-proxy
      rules:
        - runAsRootAllowed
        - notReadOnlyRootFilesystem
        - dangerousCapabilities
        - insecureCapabilities
        - livenessProbeMissing
```

however, this does not seem to be applied

![image](https://user-images.githubusercontent.com/183849/105819841-a4480c00-5fb0-11eb-927b-cc68d9cb25e5.png)

This is deployed using the helm stable release (which looks to be on version 3.0 atm)

(do you have a latest repo for helm?)

**Comment by adriankostrubiak-tomtom** [comment]
@dbones this is now available in 3.1.0
